triple-store/triple-store-impl-jena/src/main/java/com/powsybl/triplestore/impl/jena/TripleStoreJena.java

```diff
@@ -43,13 +44,16 @@
 import com.powsybl.triplestore.api.PrefixNamespace;
 import com.powsybl.triplestore.api.PropertyBag;
 import com.powsybl.triplestore.api.PropertyBags;
+import com.powsybl.triplestore.api.TripleStore;
 import com.powsybl.triplestore.api.TripleStoreException;
 
 /**
  * @author Luma Zamarreño <zamarrenolm at aia.es>
  */
 public class TripleStoreJena extends AbstractPowsyblTripleStore {
 
+    static final String NAME = "jena";
```

Comment:
This should probably be declared as private

Reply:
is package-private